nats-server/configuration/securing_nats/authorization.md

```diff
@@ -2,31 +2,30 @@
 
-You configure authorization by creating a `permissions` entry in the `authorization` object.
+A special field inside the authorization map is `default_permissions`. When present, it contains permissions that apply to users that do not have permissions associated with them.
 
 ## Permissions Configuration Map
 
 The `permissions` map specify subjects that can be subscribed to or published by the specified client.
 
 | Property | Description |
 | :--- | :--- |
-| `publish` | subject, list of subjects, or permission map the client can publish |
-| `subscribe` | subject, list of subjects, or permission map the client can subscribe |
-| `allow_responses` | boolean or object |
-
+| `publish` | subject, list of subjects, or [permission map](#permission-map) the client can publish |
+| `subscribe` | subject, list of subjects, or [permission map](#permission-map) the client can subscribe to. In this context it is possible to provide an optional queue name: `<subject> <queue>` to express queue group permissions. These permissions can also use wildcards such as `v2.*` or `>`.|
+| `allow_responses` | boolean or [responses map](#allow-responses-map), default is `false` |
 
 ## Permission Map
 
-The `permission` map provides additional properties for configuring a `permissions` map. Instead of providing a list of subjects that are allowed, the `permission` map allows you to explicitly list subjects you want to`allow` or `deny`:
+The `permission` map provides additional properties for configuring a `permissions` map. Instead of providing a list of allowable subjects and optional queues, the `permission` map allows you to explicitly list those you want to`allow` or `deny`. Both lists can be provided. In case of overlap `deny` has priority.
 
 | Property | Description |
 | :--- | :--- |
 | `allow` | List of subject names that are allowed to the client |
 | `deny` | List of subjects that are denied to the client |
 
-**Important Note** NATS Authorizations can be _allow lists_, _deny lists_, or both. It is important to not break request/reply patterns. In some cases \(as shown below\) you need to add rules as above with Alice and Bob for the `_INBOX.>` pattern. If an unauthorized client publishes or attempts to subscribe to a subject that has not been _allow listed_, the action fails and is logged at the server, and an error message is returned to the client.
+**Important Note** It is important to not break request/reply patterns. In some cases \(as shown below\) you need to add rules as above with Alice and Bob for the `_INBOX.>` pattern. If an unauthorized client publishes or attempts to subscribe to a subject that has not been _allow listed_, the action fails and is logged at the server, and an error message is returned to the client. The [allow responses](#allow-responses-map) option can simplify this.
```

Comment:
There is no Alice and Bob example show 'above'

Reply:
Thanks for finding this. Removed it and added link for below